lib/elixir/lib/map.ex

```diff
@@ -1,6 +1,21 @@
 defmodule Map do
   @moduledoc """
   A set of functions for working with maps.
+  
+  Please note many functions for maps can be found in `Enum`.
```

Comment:
How about something like: "maps implement the `Enumerable` protocol and thus can be manipulated using functions from the `Enum` module.